spec/rubocop/cop/style/redundant_string_escape_spec.rb

```diff
@@ -279,6 +279,14 @@ def wrap(contents)
     end
   end
 
+  context 'when using character literals (e.g. `?a`)' do
+    it 'does not register an offense for escaped spaces' do
```

Comment:
"escaped spaces" doesn't seem to be used.

Reply:
Oops, indeed it is. Thank you very much.